sr/robot/arenas/caldera_arena.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import division
+
+from math import ceil, cos, pi, sin
+
+import pygame
+
+from .arena import ARENA_FLOOR_COLOR, ARENA_MARKINGS_COLOR, ARENA_MARKINGS_WIDTH, Arena, draw_corner_zones
+from ..markers import Token
+
+
+# Perform a clockwise rotate around 0,0.
+def rotate(x, y, radians):
+    return (x * cos(radians) + y * -sin(radians)), (x * sin(radians) + y * cos(radians))
+
+
+PLATFORM_COLOUR = (0x68, 0x75, 0x91)
+
+class CalderaArena(Arena):
+    start_locations = [(-3.6, -3.6),
+                       (3.6, -3.6),
+                       (3.6, 3.6),
+                       (-3.6, 3.6)]
+
+    start_headings = [0.25 * pi,
+                      0.75 * pi,
+                      -0.75 * pi,
+                      -0.25 * pi]
+
+    def __init__(self, objects=None, wall_markers=True):
+        super(CalderaArena, self).__init__(objects, wall_markers)
+        self._init_tokens()
+
+    def _init_tokens(self):
+        token_ids = [28, 29, 30, 31]
+
+        token_locations_offsets_from_zone = [(0.5, 1.3),
+                                             (0.5, 1.5),
+                                             (0.5, 1.7),
+                                             (0.5, 1.9),
+                                             (0.5, 2.1),]
+
+        for zone in range(0, 4):
+            for i, location in enumerate(token_locations_offsets_from_zone):
+                rotated_location = rotate(location[0] - 4, location[1] - 4, (pi / 2) * zone)
+                token = Token(self, token_ids[zone], damping=0.5)
```

Comment:
I think we need quite a bit more damping on the tokens; my testing (just running the `test.py`) shows that they slide a lot more than I'd expect.

Reply:
Oops good catch, that's because I used capture the flag (when the tokens were on wheels) as the base

Reply:
fixed in 2f5fa9bf3ff8cd10c3ae883f1ac22ed9168cf378